tox.ini

```diff
@@ -12,10 +12,10 @@ envlist =
     py
 
 [testenv]
-whitelist_externals = 
-    pytest
```

Comment:
Why this change?

Reply:
This enables `tox` to use the purest within the original environment rather than installing (a potentially different version) within its own environment. Avoids confusion.